pkg/printers/internalversion/describe_test.go

```diff
@@ -1664,9 +1664,25 @@ Spec:
     From Pod Selector: id=app2,id2=app3
     From Namespace Selector: id=app2,id2=app3
     From Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
+    From IPBlock:
+        CIDR: 192.168.0.0/16
+        Except: 192.168.3.0/24, 192.168.4.0/24
     ----------
     To Port: <any> (traffic allowed to all ports)
     From: <any> (traffic not restricted by source)
+  Allowing egress traffic:
+    From Port: 80/TCP
```

Comment:
I think this is still `To Port:` - NP egress doesn't say anything about the source port (which is going to be a random high numbered port anyway) - this is still the destination's port.

Reply:
@caseydavenport thanks! I misunderstood it :)

Reply:
Nice catch :)

Reply:
Done. Thanks!